shopify-client/src/views/ProductUpdate.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="center">
+    <p v-if="error">{{ error }}</p>
+    <ProductForm
+      v-if="product"
+      v-bind:handleSubmit="handleSubmit"
+      v-bind:initialValues="product"
+    />
+  </div>
+</template>
+
+<script>
+import ProductForm from "@/components/ProductForm.vue";
+
+export default {
+  name: "ProductUpdate",
+  props: ["id"],
+  components: {
+    ProductForm,
+  },
+  data: function() {
+    return {
+      product: null,
+      error: null,
+    };
+  },
+  created: function() {
+    const vm = this;
+    fetch(`/api/products/${vm.id}`)
+      .then((response) => response.json())
+      .then((response) => {
+        if (response.status == 404) {
```

Comment:
_Nitpick_: Use ===

Reply:
The == is done on purpose. The status in the response is a string field. Otherwise I would need to do === "404". In this case, I was taking advantage of the type-ignorant comparison.